config-intelliJ.sh

```diff
@@ -0,0 +1,9 @@
+if [ "$1" = "java8" ]; then
```

Comment:
@elrodro83 , Martin's opinion is that this script should be here. I think it should go in mule-uber/bin as all the other scripts. What say you?

Reply:
vote for `mule-uber/bin`, as all other tools used to set up the workspace